infra/terraform/test_fixtures/terraform-google-vm.tf

```
@@ -22,16 +22,16 @@ resource "google_project_services" "vm" {
 
   services = [
     "compute.googleapis.com",
-    "storage-api.googleapis.com",
+    "iam.googleapis.com"
   ]
 }
 
 resource "google_service_account" "vm" {
   provider = "google.phoogle"
 
   project      = "${google_project.vm.id}"
-  account_id   = "phoogle-ci-vm-id"
-  display_name = "phoogle-ci-vm-id"
+  account_id   = "ci-vm"
```

This account ID is invalid; account IDs are required to be at least 6 characters long:

```
Error: google_service_account.vm: "account_id" ("ci-vm") doesn't match regexp "^[a-z](?:[-a-z0-9]{4,28}[a-z0-9])$"
``` 

```suggestion
  account_id   = "ci-google-vm"
```

---

Thank you for your review. Done.